packages/Spinner/__tests__/Spinner.spec.jsx

```diff
@@ -50,6 +50,14 @@ describe('Spinner', () => {
     expect(spinner).toMatchSnapshot()
   })
 
+  it('renders inline', () => {
+    const { spinner } = doMount(
+      { spinning: true, inline: true },
+      <span>Overlay me with the spinner</span>
+    )
+    expect(spinner.childAt(0)).toHaveClassName('inline')
```

Comment:
Could this be changed to use a `data-testid`? This could break suddenly if we ever change the DOM at all.